src/opendr/planning/end_to_end_planning/__init__.py

```diff
@@ -1,4 +1,6 @@
 from opendr.planning.end_to_end_planning.e2e_planning_learner import EndToEndPlanningRLLearner
-from opendr.planning.end_to_end_planning.envs.UAV_depth_planning_env import UAVDepthPlanningEnv
+import os
+if os.environ.get("ROS_VERSION") == "1":
```

Comment:
```suggestion
if os.environ.get("ROS_DISTRO") == "melodic" or os.environ.get("ROS_DISTRO") == "noetic":
```

Reply:
I don't believe ROS_VERSION is present out of the box, and we don't want to add another environmental variable considering it can be inferred from ROS_DISTRO. Moreover, we only really support melodic and noetic so I think this is better.